trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'decorator'
+
+class TrimmerDecorator
+  def initialize(nameable)
+    # super(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name.length > 10 ? @nameable.correct_name[0...10] : @nameable.correct_name
+  end
+end
```

Comment:
- [ ] [Optional] I would suggest that you inherit the `TrimmerDecorator` class from the `Decorator` class as you did for the  `CapitalizeDecorator` class.

Reply:
Hey @thecodechaser, thanks for your suggestion, I'll do that 😄